modules/@apostrophecms/schema/ui/apos/components/AposInputRelationship.vue

```diff
@@ -181,21 +196,8 @@ export default {
     updateSelected(items) {
       this.next = items;
     },
-    async input () {
-      if (this.searching) {
-        return;
-      }
-
-      const trimmed = this.searchTerm.trim();
-      if (!trimmed.length) {
-        this.searchList = [];
-        return;
-      }
-
-      const qs = {
-        autocomplete: trimmed
-      };
-
+    async search(qs = {}) {
+      qs.perPage = this.field.suggestionLimit;
```

Comment:
I don't see where suggestionSort is applied.

Reply:
We have the following in piece type https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/piece-type/index.js#L1067

```
      find(_super, req, criteria, projection) {
        return _super(req, criteria, projection).defaultSort(self.options.sort || { updatedAt: -1 });
      }
```

IMHO we can get away with the default sort. (I forgot to delete the option)